staging/src/k8s.io/apiextensions-apiserver/pkg/controller/establish/establishing_controller.go

```diff
@@ -135,6 +135,10 @@ func (ec *EstablishingController) sync(key string) error {
 
 	// Update server with new CRD condition.
 	_, err = ec.crdClient.CustomResourceDefinitions().UpdateStatus(crd)
+	if apierrors.IsNotFound(err) || apierrors.IsConflict(err) {
+		// deleted or changed in the meantime, we'll get called again
```

Comment:
regarding "we'll get called again", returning nil will result in no call anymore for this CRD, right?

Reply:
returning nil means we don't requeue *because of an error*. if the CRD is not found, we don't want to requeue. if we got a conflict, that means another event occurred, which our informer will observe and queue.

Reply:
nit: would it be more clear to say "we'll get called again on further events" or something like this. The "we'll get called again" gave me the `requeue` impression.

Reply:
I don't feel that strongly either way